.github/workflows/end-to-end.yml

```
@@ -222,27 +209,13 @@ jobs:
       with:
         args: route53 change-resource-record-sets --hosted-zone-id Z2STJRQSJO5PZ0 --change-batch file://apps/pipeline/config/exoscale/dns-${{ matrix.cluster }}.json
 
-  buckets-cleanup:
-    needs: apps
-    if: always()
-    runs-on: ubuntu-20.04
-    steps:
-    - name: Checkout bucket-manager
-      uses: actions/checkout@v2
-      with:
-        repository: elastisys/bucket-manager
-        ref: main
-        token: ${{ secrets.CK8S_PIPELINE_GITHUB_TOKEN }}
     - name: Delete buckets
+      if: matrix.cluster == 'sc'
```

I think you need to add `always()` or it won't run if there was an error earlier.
```suggestion
      if: always() && (matrix.cluster == 'sc')
```